libs/fixtures/data/electionComplexGeoSample/README.md

```diff
@@ -0,0 +1,8 @@
+### Fixture Purpose
```

Comment:
Not sure where else to provide this context, which I think is relevant to a reader of the code.